### PR TITLE
Added changes to kebechet model

### PR DESCRIPTION
**Pull request by saisankargochhayat**
## Related Issues and Dependencies
Added the necessary columns to store the enabled managers and the timestamp when kebechet was last run. 

Related - #2016

### CODE REVIEW

File: thoth/storages/graph/models.py [hunk 1]
```diff
@@ -1447,6 +1447,15 @@ class KebechetGithubAppInstallations(Base, BaseExtension):
...
     is_active = Column(Boolean, nullable=False)
+    # Kebechet managers activated per repo
+    info_manager = Column(Boolean, nullable=False, default=False)
+    pipfile_requirements_manager = Column(Boolean, nullable=False, default=False)
+    update_manager = Column(Boolean, nullable=False, default=False)
+    version_manager = Column(Boolean, nullable=False, default=False)
+    thoth_advise_manager = Column(Boolean, nullable=False, default=False)
+    thoth_provenance_manager = Column(Boolean, nullable=False, default=False)
+    # Last run time-stamp
+    last_run = Column(DateTime(timezone=False), nullable=False)
```
Comment (fridex) [inline thoth/storages/graph/models.py:1458]: I think a nullable field can be acceptable here (meaning not run yet).

**Review by fridex — COMMENTED** [review]
A migration will be needed. Otherwise LGTM :+1:

**Comment by saisankargochhayat** [comment]
@pacospace @fridex Added the migration and tested locally :)

**Review by fridex — APPROVED** [review]
lgtm :100:

**Pull request merged**